modules/mysql/src/main/resources/META-INF/services/org.testcontainers.containers.JdbcDatabaseContainerProvider

```diff
@@ -1 +1 @@
-org.testcontainers.containers.MySQLContainerProvider
\ No newline at end of file
+org.testcontainers.containers.MySQLContainerProvider
```

Comment:
rollback this change. it should not appear in the PR

Reply:
OK